zingolib/src/wallet/keys/unified.rs

```diff
@@ -15,7 +15,7 @@ use sapling_crypto::note_encryption::SaplingDomain;
 use zcash_primitives::consensus::{BranchId, NetworkConstants, Parameters};
 
 use crate::config::{ChainType, ZingoConfig};
-use secp256k1::SecretKey;
+use secp256k1::{PublicKey as Secp256k1PublicKey, SecretKey};
```

Comment:
if my suggestion below is followed, this alias should be removed. in fact its best to remove this whole line and also call SecretKey as `secp256k1::SecretKey` everywhere it's called.